config/pure-python-without-pypy/tox.ini

```diff
@@ -0,0 +1,40 @@
+[tox]
+envlist = flake8,
+          py27,
+          py35,
+          py36,
+          py37,
+          py38,
+          coverage
+
+[testenv]
+usedevelop = true
+commands =
+    zope-testrunner --test-path=src []
+extras = test
+
+[testenv:flake8]
+basepython = python3
+skip_install = true
+deps = flake8
+commands = flake8 src setup.py
+
+[testenv:coverage]
+basepython = python3
+deps =
+    coverage
+commands =
+    coverage run -m zope.testrunner --test-path=src []
+    coverage html
+    coverage report
+
+[coverage:run]
```

Comment:
Does this coverage configuration get picked up? I would expect that to be in setup.cfg. If it gets read from here that's fine, I am just wondering.

Reply:
Yes, it should according to https://coverage.readthedocs.io/en/v4.5.x/config.html#configuration-files (last paragraph of first section).